ginga/AstroImage.py

```diff
@@ -62,7 +61,21 @@ def __init__(self, data_np=None, metadata=None, logger=None,
 
         self.io = ioclass(self.logger)
 
+        # these attributes to be removed--DO NOT USE!!!
+        inherit_primary_header = False
```

Comment:
BTW, as shown here, the `inherit_primary_header` and `save_primary_header` get the same default values (False and True, respectively) as they had previously.